frontend/src/auth/auth.spec.js

```diff
@@ -1,6 +1,6 @@
-import { auth } from '@/auth'
```

Comment:
Ich glaube diese Datei könntest du noch in einen `__tests__` Unterordner verschieben, so wie die Tests von Smiley